metrics-httpclient/src/test/java/io/dropwizard/metrics/httpclient/InstrumentedHttpClientsTest.java

```diff
@@ -10,6 +16,8 @@
 import io.dropwizard.metrics.MetricRegistry;
 import io.dropwizard.metrics.MetricRegistryListener;
 import io.dropwizard.metrics.Timer;
+import static junit.framework.TestCase.fail;
```

Comment:
I think it's better to use `import static org.junit.Assert.fail` from the consistency point of view, because we already import `org.junit.Before` and `org.junit.Test`